krabs/krabs/ut.hpp

```diff
@@ -113,19 +122,20 @@ namespace krabs { namespace details {
         // TODO: Only forward the calls that are requested to each provider.
         for (auto &provider : trace.providers_) {
             if (provider_flags.find(provider.get().guid_) != provider_flags.end()) {
```

Comment:
Re-reading this, the `if` block here isn't needed at all. Just remove the whole thing.
To initialize the settings' `filter_flags_` member, add a agg-initializer to line 34 on the struct member `filter_flags filter_flags_{};` and you should be good.